.changes/unreleased/tinymce-TINY-11335-2024-10-08.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Improved
+body: Elements with `data-mce-resize` set to true are now resizable.
```

Comment:
```suggestion
body: All elements with `data-mce-resize` set to true are now resizable.
```

Reply:
I wonder if we should hide this it's really not a documented feature also if we document it then we kind of need to support the way it works today. It also a editor internal temporary attribute so explaining all that would require a docs section on it's own. Also it's oddly named I really should have named it `data-mce-resizable` 😄

Reply:
yes, I forgot to update this after my other comment

- [ ] delete changelog entry